RabbitMQ.Stream.Client/Message.cs

```diff
@@ -36,9 +65,55 @@ public ReadOnlySequence<byte> Serialize()
 
         public static Message From(ReadOnlySequence<byte> amqpData)
         {
+            //                                                         Bare Message
+            //                                                             |
+            //                                       .---------------------+--------------------.
+            //                                      |                                           |
+            // +--------+-------------+-------------+------------+--------------+--------------+--------
+            // | header | delivery-   | message-    | properties | application- | application- | footer |
+            // |        | annotations | annotations |             | properties  | data         |        |
+            // +--------+-------------+-------------+------------+--------------+--------------+--------+            
+
```

Comment:
May I suggest to add these comments? I think they are useful:
 //  Altogether a message consists of the following sections:
            // • Zero or one header.
            // • Zero or one delivery-annotations.
            // • Zero or one message-annotations.
            // • Zero or one properties.
            // • Zero or one application-properties.
            // • The body consists of either: one or more data sections, one or more amqp-sequence sections,
            // or a single amqp-value section.
            // • Zero or one footer.